srcpkgs/makemkv/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'makemkv'
+pkgname=makemkv
+version=1.15.4
+revision=1
+archs="x86_64"
```

Comment:
Seems like I forgot to reply here: Yes, it is, because it's not *fully* built from source. They ship some proprietary libs, see the first of the two distfiles..